pkg/deploy/deploy.go

```diff
@@ -151,3 +156,14 @@ type networkError struct {
 func (d *networkError) Error() string {
 	return "Unable to reach Armory Continuous Delivery as A Service. Please check your internet connection and try again. "
 }
+
+func getPipelinePath(options StartPipelineOptions) (string, error) {
+	structured, err := options.structuredConfig()
+	if err != nil {
+		return "", err
+	}
+	if structured.Kind == "kubernetes" || structured.Kind == "" {
+		return "/pipelines/kubernetes", nil
+	}
+	return "/pipelines", nil
+}
```

Comment:
Yeah, I like this. Lets not make the CLI need to know about the path for each new kind we add. 👍